app/institutions/index/styles.scss

```diff
@@ -105,7 +105,7 @@
                     background-color: $color-white;
                     box-shadow: 1px 1px 1px $color-gray-light;
                     padding: 6px;
-                    color: $color-text-gray;
```

Comment:
Did you happen to run AXE tools over some of these screens after you changed the text color? The color you changed it to is darker, so should be fine from an accessibility standpoint, but changing text color is the riskiest part of this PR that I can think of.